wp-content/themes/core/blocks/icongrid/icongrid.php

```diff
@@ -0,0 +1,9 @@
+<?php
+declare( strict_types=1 );
```

Comment:
Move this to top line in each file, will make future PHPCS stuff easier for sure.